_posts/2021-04-02-cultural-competency-in-archives-planning-an-education-campaign.md

```diff
@@ -0,0 +1,69 @@
+
+## ...With A Focus on Education
+Our education campaign aimed to formulate and facilitate training activities to equip staff with the knowledge and expertise to recognize, prevent and remove pejorative, racist, xenophobic, homophobic, transphobic, sexist, euphemistic, and ableist language; hagiographic description that valorizes white men and people with power; and to address the absence of description related to individuals and groups from marginalized communities.
+
+Preparing to take full responsibility for running this campaign, Katie, Darren, and I met with other staff members who had experience running an education program. We were able to learn from their experiences and gather helpful resources. We wanted to make sure participants understood that this was a group learning experience and even though we would act as learning facilitators, we were not experts. We gathered and shared specific resources that were geared toward both personal and institutional change that would be extremely important as we moved through the education campaign.
+
+Below is an excerpt from the Culturally Competent (Re)description Education Campaign that shows how we plan to navigate learning together what it means to be “culturally competent”.
+
+### Education Campaign Project Milestones
+**Phase 1: Self-Guided Remote Learning**
+
+Goals:
+- Introduce cultural competency concepts and practices in the framework of archives
+- Create awareness of unconscious bias and racism within ourselves and our collections
+- Identification of inclusive and reparative description in archives
+
+Proposed Courses and Learning Modules:
+- LinkedIn Learning [Confronting Racism Course](https://www.linkedin.com/learning/confronting-racism-with-robin-diangelo)
+- SAA [Cultural Diversity Competency Course](https://www2.archivists.org/prof-education/course-catalog/cultural-diversity-competency)
+- SAA Archives*Records 2020, On-Demand Session 24: [Implementing Inclusive (Re)Description at Predominantly White Institutions](https://mvp.markeys.onl/SAA/agenda/session/306593)
+- Three discussion sessions in January 2021 to accommodate staff
+
+**Phase 2: Reading Group**
+
+Goals:
+- Monthly virtual meetings with co-facilitated discussions via Teams/Zoom 
+- Establish and share ground rules for productive and safe discussion
+- Meetings begin with a lightning round style reflection question that is shared with staff prior to the meeting
+
+Recommended Readings: 
+- [Silencing the Past, Chapter 1: The Power in the Story](https://prismatically.blog/2020/08/20/book-summary-silencing-the-past-michel-rolph-trouillot/#:~:text=Chapter%201%3A%20The%20Power%20in,is%20said%20to%20have%20happened'.&text=Some%2C%20by%20being%20influenced%20by,and%20the%20narratives%20about%20it.) - Michel-Rolph Trouillot
```

Comment:
Is it possible to clean this link up a bit?

Reply:
A `bit.ly` link might be a good idea.

Reply:
https://prismatically.blog/2020/08/20/book-summary-silencing-the-past-michel-rolph-trouillot/ works fine

Reply:
i used a bit.ly because i wanted it to link to the Chapter 1 heading. happy to change it, i'm not committed to it